cob_frame_tracker/src/cob_frame_tracker.cpp

```diff
@@ -330,18 +306,48 @@ void CobFrameTracker::publishTwist(ros::Duration period, bool do_publish)
         twist_pub_.publish(twist_msg);
 }
 
+void CobFrameTracker::publishHoldTwist(const ros::Duration& period)
+{
+    geometry_msgs::TwistStamped twist_msg;
+    twist_msg.header.frame_id = chain_tip_link_;
+
+    if(!this->ht_.hold)
+    {
+        ROS_ERROR_STREAM_THROTTLE(1, "Abortion active: Publishing zero twist");
```

Comment:
Did you verify that the THROTTLE really works? I'm not sure but I had in mind that streaming throttles down the whole node. Be careful. This might result in lower publishing rates!